src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/histogram.js

```diff
@@ -9,16 +9,16 @@ let shortcuts = {
     debugPreprocessor: e => (e.ctrlKey || e.metaKey) && e.key === 'd',
 }
 
-const LABEL_STYLE = 'font-family: DejaVuSansMonoBook; font-size: 10px;'
-const MARGIN = 25
-const X_AXIS_LABEL_WIDTH = 10
-const Y_AXIS_LABEL_WIDTH = 10
-const ANIMATION_DURATION = 1000
-const LINEAR_SCALE = 'linear'
-const LIGHT_PLOT_COLOR = '#00E890'
-const DARK_PLOT_COLOR = '#E0A63B'
-const DEFAULT_NUMBER_OF_BINS = 10
-const BUTTON_HEIGHT = 25
+const labelStyle = 'font-family: DejaVuSansMonoBook; font-size: 10px;'
+const margin = 25
+const xAxisLabelWidth = 10
+const yAxisLabelWidth = 10
+const animationDuration = 1000
+const linearScale = 'linear'
+const lightPlotColor = '#00E890'
+const darkPlotColor = '#E0A63B'
+const defaultNumberOfBins = 10
+const buttonHeight = 25
```

Comment:
why this change? We are using upper case for constants everywhere

Reply:
Is this the `pretty` way?
I think we are actually missing linting for these JS files and there is an old PR of mine in limbo, which would have extended this.